### ISSUE TITLE
Next steps?

### ISSUE THREAD

**Issue by guilherme** [sebglazebrook/conch#6]
What could be the next steps for the language?
- Evaluating other operations?
- Evaluating expressions inside of parameters. ( 1 + 1 ) + 1?
- Evaluating variable assignment?
- Ideas?

**Comment by sebglazebrook** [comment]
Yeah why don't we do the other main types of calculator functions and then add order of operation stuff. Like using brackets etc. 

Variable assign will be good but only when we can evaluate multi line code.

**Comment by sebglazebrook** [comment]
Also should we list out what is required to be considered turing complete and go through them one by one?

**Comment by sebglazebrook** [comment]
@guilherme ^^

**Comment by guilherme** [comment]
sounds  good! this idea of turing complete!

**Comment by guilherme** [comment]
by the way, i receive the notifications, but nothing pops up for me. I'll need to set this up
